### ISSUE TITLE
Incorrect position of the function that add style class 

### ISSUE THREAD

**Issue by mapirelli** [sandywalker/webui-popover#102]
Yuo must move the adding of the style class before the calculation of the height and width of the popover, in order to avoid a ugly bug on absolute position calculation.

Mariano Pirelli.   

            //TODO Move here
            if (this.options.style) {
                $target.addClass(pluginClass + '-' + this.options.style);
            }

            //init the popover and insert into the document body
            if (!this.options.arrow) {
                $target.find('.arrow').remove();
            }
            $target.detach().css({
                top: -2000,
                left: -2000,
                display: 'block'
            });
            if (this.getAnimation()) {
                $target.addClass(this.getAnimation());
            }
            $target.appendTo(document.body);
            targetWidth = $target[0].offsetWidth;
            targetHeight = $target[0].offsetHeight;
            placement = this.getPlacement(elementPos);
            this.initTargetEvents();
            var postionInfo = this.getTargetPositin(elementPos, placement, targetWidth, targetHeight);
            this.$target.css(postionInfo.position).addClass(placement).addClass('in');

            if (this.options.type === 'iframe') {
                var $iframe = $target.find('iframe');
                $iframe.width($target.width()).height($iframe.parent().height());
            }
            //TODO Remove from here
/*
            if (this.options.style) {
                this.$target.addClass(pluginClass + '-' + this.options.style);
            }
*/

**Comment by sandywalker** [comment]
Sorry for replay so late, I've updated the code to fix this issue , Thanks.

**Comment by mapirelli** [comment]
Nope, 
Thank you for the work you do. Very good plugin.
